src/app/public/styles/ag-grid-styles.scss

```diff
@@ -32,6 +32,13 @@ ag-grid-angular:not(.sky-ag-grid-editable) {
 
   border-top: $sky-standard-border;
 
+  .ag-cell.ag-cell-wrap-text {
+    min-height: ag-param(row-height) - 2px;
+    line-height: normal;
```

Comment:
We discussed offline about changing the line heights to `$sky-line-height-base` for default theme and then `$sky-theme-modern-font-paragraph-line-height` for modern